integration-tests/pipelines/e2e-tests-kind-pipeline.yaml

```diff
@@ -116,7 +116,9 @@ spec:
         - name: id
           value: $(context.pipelineRun.name)
         - name: tags
-          value: env=konflux,user=release-service-catalog
+          value: >
+            'owner=konflux-devprod@redhat.com,project=Konflux,created-by=integration-pipeline,
+            iac=mapt,cluster-name=$(context.pipelineRun.name),k8s-type=kind,component=release-service-catalog'
```

Comment:
For [#486](https://github.com/konflux-ci/build-service/pull/486), I see that it also required the projectName value to be provided. At least this is what I see in the description.
It can be easily added to this list in case the answer is positive:
projectName=$(context.pipelineRun.namespace)